docs/pages/api-reference/sources/mongo.md

```diff
@@ -0,0 +1,71 @@
+---
+title: MongoDB
+order: 0
+status: published
+---
+### MongoDB
+Data connector to MongoDB databases.
+
+#### Database Parameters
+<Expandable title="name" type="str">
+A name to identify the source. The name should be unique across all Fennel sources.
+</Expandable>
+
+<Expandable title="host" type="str">
+The hostname of the database.
+</Expandable>
+
+<Expandable title="db_name" type="str">
+The name of the Mongo database to establish a connection with.
+</Expandable>
+
+<Expandable title="username" type="str">
+The username which should be used to access the database. This username should 
+have access to the database `db_name`.
+</Expandable>
+
+<Expandable title="password" type="str">
+The password associated with the username.
+</Expandable>
+
+:::info
+We use [SRV connection format](https://www.mongodb.com/docs/manual/reference/connection-string/#std-label-connections-dns-seedlist) 
+for authentication which is supported in Mongo versions 3.6 and later. If you have 
+a self-hosted DB with version earlier than 3.6, please reach out to Fennel support
+:::
+
+#### Table Parameters
+<Expandable title="table" type="str">
+The name of the table within the database that should be ingested.
+</Expandable>
+
+<Expandable title="cursor" type="str">
+The name of the field in the table that acts as `cursor` for ingestion i.e. 
+a field that is approximately monotonic and only goes up with time. 
+
+Fennel issues queries of the form `select * from table where {cursor} >= {last_cursor - disorder}`
```

Comment:
We should mongo specific query instead of sql-like query here?

Reply:
Updated the query to mongo specific